inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/user/UserServiceImpl.java

```diff
@@ -355,10 +355,7 @@ public void login(UserLoginRequest req) {
     }
 
     public void checkUser(String inCharges, String user, String errMsg) {
```

Comment:
Does this method need anymore?

Reply:
No need anymore, it will be removed, thx.